### ISSUE TITLE
goss blows up if command exit-status is misnamed

### ISSUE THREAD

**Issue by shalomb** [aelsabbahy/goss#71]
Took me quite a while to figure out what was wrong here

```
$ cat bad.yaml
---
service:
  sshd:
    enabled: true
    running: true
command:
  uname -a:
    exit-code: 0
```

```
$ goss -g bad.yaml v
panic: Unexpected matcher type: <nil>

goroutine 6 [running]:
github.com/aelsabbahy/goss/resource.matcherToGomegaMatcher(0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/gomega.go:28 +0x333
github.com/aelsabbahy/goss/resource.ValidateValue(0x7f8bcf45dab8, 0xc82004ec40, 0x7d5bf0, 0xb, 0x0, 0x0, 0x6dcc80, 0xc82000f700, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/validate.go:66 +0x3f9
github.com/aelsabbahy/goss/resource.(*Command).Validate(0xc82004ec40, 0xc820078640, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/resource/command.go:38 +0x350
github.com/aelsabbahy/goss.Validate.func2(0xc820013e00, 0xc82000b320, 0xc82000b2c0, 0xc820078640)
        /home/travis/gopath/src/github.com/aelsabbahy/goss/validate.go:69 +0xb3
created by github.com/aelsabbahy/goss.Validate
        /home/travis/gopath/src/github.com/aelsabbahy/goss/validate.go:72 +0x471

... output truncated
```

I then generated a yaml file using `autoadd` to diff this file .. turns out I wanted to use `exit-status` not `exit-code`

```
diff -Naur bad.yaml good.yaml                 
--- bad.yaml    2016-04-08 16:06:38.210776099 +0200
+++ good.yaml   2016-04-08 16:14:38.606694865 +0200
@@ -1,8 +1,11 @@
 ---
+
 service:
   sshd:
     enabled: true
     running: true
+
 command:
   uname -a:
-    exit-code: 0
+    exit-status: 0
+
```

So a clear `PEBKAC` error :) but please can we make `goss` handle invalid input a bit better and give the user a clearer error message? This took me a good `45` minutes to figure out.

**Comment by aelsabbahy** [comment]
Sorry for the inconvenience, this is definitely not desired and looks like it was introduced sometime during the gomega changes.

Thanks for filing this.

**Comment by aelsabbahy** [comment]
Fixed in [v0.1.5](https://github.com/aelsabbahy/goss/releases/tag/v0.1.5).
